snap/snapcraft.yaml

```diff
@@ -99,6 +100,7 @@ parts:
       - kodi-wayland
       - kodi-visualization-spectrum
       - kodi-repository-kodi
+      - try: [kodi-inputstream-adaptive] # only available for x86
```

Comment:
I've a mild preference for saying exactly what should work over trying and ignoring failures, but :man_shrugging: